api/v1alpha1/bundle_webhook.go

```diff
@@ -34,7 +34,7 @@ func (r *Bundle) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-//+kubebuilder:webhook:path=/validate-core-rukpak-io-v1alpha1-bundle,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.rukpak.io,resources=bundles,verbs=create;update,versions=v1alpha1,name=core.rukpak.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/validate-core-rukpak-io-v1alpha1-bundle,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.rukpak.io,resources=bundles,verbs=create;update,versions=v1alpha1,name=vbundles.core.rukpak.io,admissionReviewVersions=v1
```

Comment:
I'm going commit-by-commit when reviewing this PR, so it's possible I'm looking at a backwards facing snapshot: is the `vbundles*` change a typing mistake? Should it be `name=bundles.core.rukpak.io` instead?

Reply:
No that was intentional. I took a hint from the way kubebuilder scaffolds webhooks. It uses `v` and `m` prefixes for validating and mutating webhooks respectively.